http_service/tests/test_bug_classification.py

```diff
@@ -158,6 +158,44 @@ def do_request():
         "bugs": {str(bug_id): result for bug_id in bug_ids}
     }
 
+    ## Test for missing Bug Ids
+    # bug and no_bug list
+    bugs_list = [1602463, 1619699]
+    missing_bugs_list = [1598744, 1615281, 1566486]
+    # merge the two list
+    bug_ids = [*bugs_list, *missing_bugs_list]
+    # A call to Buzgilla will return lesser values
+    # Five bug Ids but returns just 2 bug Ids
+    responses.add(
+        responses.GET,
+        "https://bugzilla.mozilla.org/rest/bug?id=1566486%2C1598744%2C1602463%2C1615281%2C1619699&include_fields=id&include_fields=last_change_time",
```

Comment:
```suggestion
        "https://bugzilla.mozilla.org/rest/bug?id=1566486,1598744,1602463,1615281,1619699&include_fields=id&include_fields=last_change_time",
```

Please replace `%2C` with `,`